src/zenml/zen_server/routers/auth_endpoints.py

```diff
@@ -499,9 +482,7 @@ def api_token(
     pipeline_id: Optional[UUID] = None,
     schedule_id: Optional[UUID] = None,
     expires_minutes: Optional[int] = None,
-    auth_context: AuthContext = Security(
-        authorize, scopes=[PermissionType.WRITE]
-    ),
+    auth_context: AuthContext = Security(authorize),
```

Comment:
This endpoint can be used by any user or service account regardless of their permissions to generate an API token. I guess it's harmless, because the user can already authenticate to the server to be even allowed to call the endpoint, but just wanted to point it out.

Reply:
The generated token will have the same permissions as the user that generated it, correct? In other words, when someone authenticates with this API token the user in the auth context is the same as the one that generated the token?

Reply:
Yes, correct.

Reply:
The endpoint is currently only used to generate API tokens to run pipelines though. I guess we could condition it on the pipeline run CREATE action, for the lack of a better thing.